src/contribute/contribution-guide.md

```diff
@@ -23,7 +23,7 @@ We use a review-then-commit workflow in Beam for all contributions.
 
 **For all contributions:**
 
-1. **Code:** The best part ;-)
+1. **Code:** The best part ;-) When developing a new `PTransform`, consult the [PTransform Style Guide]({{ site.baseurl }}/documentation/ptransform-style-guide).
```

Comment:
This is a list of steps. I'd prefer to keep guidance out of it the list itself. I'd suggest moving to the `Code` section below, just above `One-time Setup`.